gradle/libs.versions.toml

```diff
@@ -0,0 +1,18 @@
+[versions]
+axion-release-plugin = "1.15.5"
+nexus-publish-plugin = "1.1.0"
```

Comment:
we can use 1.3.0

Reply:
I was looking at the [wrong repo](https://mvnrepository.com/artifact/io.github.gradle-nexus/publish-plugin?repo=imagej-public) searching for the latest version 😅